src/background/index.ts

```diff
@@ -43,6 +45,7 @@ if (WATCH) {
                     break;
                 }
                 case 'reload:full': {
+                    tabs.forEach((tabId) => chrome.tabs.sendMessage(tabId, {type: 'reload'}));
```

Comment:
It feels like we can reuse the `queryTabs` function from tab-manager.ts 

https://github.com/darkreader/darkreader/blob/02d6f7dd684e0cd6805c37a11c1908a42874abe3/src/background/tab-manager.ts#L6-L10

So we can avoid having them in a set. Especially because the `unload` event can be interpreted by the browser in many ways and have the chance of not being called at all, because it could be killed by an OOM of the operating system and many more reasons.

Reply:
I changed it to use `chrome.tabs.query` directly because there isn't much bennefit in wrapping it into a Promise just to unwrap that with a `then`. I initially stored tab ids explicitly because `chrome.tabs.query` just returns all tabs whuch exist without regard to whether there are any content scripts, which could be less efficient.

Does this look better now?

Reply:
Synchronous vs asynchronous can have a lot of benefit and ensure that certain things has been done if you rely on certain actions. But in this case I don't see any relevance so that's fine.

> just returns all tabs whuch exist without regard to whether there are any content scripts, which could be less efficient.

It's a debug-copy anyway not something to be used in production so I won't even care if it has 100 tabs whereby 1 tab is actually something with dark reader injected. 

It looks better now.